app/templates/prepare_db_for_dev.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Define the root of the GIT repository.
+cd ${0%/*}
+ROOT=$(pwd)
+cd $ROOT
+
+# Load the colors.
+source $ROOT/scripts/helper-colors.sh
+
+# Trying to get the table name from the config.sh file.
+if [ -f $ROOT/config.sh ]; then
+  source $ROOT/config.sh
+  TABLE_NAME="$MYSQL_DB_NAME"
+fi
+
+# Override table name if provided as an argument.
+if [ $1 ]; then
+  TABLE_NAME="$1"
+fi
+
+if [ ! $TABLE_NAME ]; then
+    echo
+    echo -e  "${BGRED}                                                     ${RESTORE}"
+    echo -e "${BGLRED}  ERROR >>> Please provide a table name!             ${RESTORE}"
+    echo -e  "${BGRED}     Example:                                        ${RESTORE}"
+    echo -e  "${BGRED}         > bash prepare-db-for-dev.sh <table_name>   ${RESTORE}"
+    echo -e  "${BGRED}                                                     ${RESTORE}"
+    echo
+    exit 1
+fi
+
+echo
+echo -e "${BLUE}> You're about to modify a table named: ${LYELLOW}'$TABLE_NAME'${RESTORE}"
+read -p "> Are you sure (y/n)? "
+
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    exit
+fi
+
+echo -e "${LBLUE}> Preparing your database for development.${RESTORE}"
+cd $ROOT/www
+
+echo
+echo " > Changing the solr index to be read only."
+mysql -uroot -e "UPDATE $TABLE_NAME.search_api_index SET read_only = '1'"
+
+echo
+echo " > Removing emails from all users."
+mysql -uroot -e "UPDATE $TABLE_NAME.users SET mail = 'mail@example.invalid'"
```

Comment:
lets just add `.test` suffix to all mail instead - `UPDATE `users` SET `mail`=concat(mail, '.test') WHERE `uid` > 0`